src/Microsoft.AspNetCore.Mvc.Core/Internal/ChunkingCookieManager.cs

```diff
@@ -0,0 +1,284 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
```

Comment:
This was already removed actually. Probably you have might looked at an earlier commit.